src/renderer/Renderer.ts

```diff
@@ -182,7 +205,7 @@ abstract class Renderer {
     const align = this._align;
 
     return typeof align === "object"
-      ? (align as { anchor: string | number }).anchor
+      ? (align as { panel: string | number }).panel
```

Comment:
Is there a type other than number, string, or object for align?

Reply:
Those three types that you've mentioned are only types that are used for the align option.